arcgis-ios-sdk-samples/Content Display Logic/Views/GroupLayersSectionView.xib

```diff
@@ -0,0 +1,50 @@
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="LayersTableViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="XjP-9q-BD7" customClass="GroupLayersSectionView" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="375" height="44"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <stackView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="X6R-rt-aHC">
+                    <rect key="frame" x="5" y="5" width="365" height="34"/>
+                    <subviews>
+                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="10R-ek-huY">
+                            <rect key="frame" x="0.0" y="0.0" width="316" height="34"/>
+                            <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
+                            <nil key="textColor"/>
+                            <nil key="highlightedColor"/>
+                        </label>
+                        <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="dVd-4F-fda">
+                            <rect key="frame" x="316" y="0.0" width="51" height="34"/>
+                            <connections>
+                                <action selector="toggleSwitch:" destination="-2" eventType="valueChanged" id="3hj-XH-AtL"/>
+                            </connections>
+                        </switch>
+                    </subviews>
+                </stackView>
+            </subviews>
+            <constraints>
+                <constraint firstAttribute="bottom" secondItem="X6R-rt-aHC" secondAttribute="bottom" constant="5" id="4Da-1Z-GB0"/>
+                <constraint firstAttribute="trailing" secondItem="X6R-rt-aHC" secondAttribute="trailing" constant="5" id="OI6-c5-c6y"/>
+                <constraint firstItem="X6R-rt-aHC" firstAttribute="top" secondItem="XjP-9q-BD7" secondAttribute="top" constant="5" id="mKF-H0-Gdd"/>
+                <constraint firstItem="X6R-rt-aHC" firstAttribute="leading" secondItem="XjP-9q-BD7" secondAttribute="leading" constant="5" id="wLZ-ZN-DIT"/>
```

Comment:
Please constrain the leading and trailing edges of the stack view to the superview's margins, with a constant of zero. That matches the inset of the rows themselves much more closely, albeit not exactly. But close enough.

You'll need to tick down the horizontal content compression resistance priority of the label by one, otherwise it'll push the switch off to the side:

![image](https://user-images.githubusercontent.com/2257493/56002743-5dbcdc80-5c78-11e9-887a-95cf0df3cc70.png)